src/models/utils/selectableStaticFilter.ts

```diff
@@ -1,12 +1,16 @@
-import { Filter } from '@yext/search-core';
+import { StaticFilter } from '@yext/search-core';
 
 /**
- * A {@link Filter} with additional information, such as an
+ * A {@link StaticFilter} with additional information, such as an
  * optional display name and whether or not it is selected.
  *
  * @public
  */
-export interface SelectableFilter extends Filter {
```

Comment:
You don't want to extend anymore here?

Reply:
We can't extend anymore because `StaticFilter` is a union type. TS gives this error if I try: "An interface can only extend an object type or intersection of object types with statically known members."